subsys/bluetooth/services/Kconfig.ots

```diff
@@ -0,0 +1,46 @@
+# Bluetooth GATT Object Transfer service
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_GATT_OTS
+	bool "Object Transfer Service"
+	select BT_L2CAP_DYNAMIC_CHANNEL
+	select BT_GATT_DYNAMIC_DB
+	select BT_SMP
+	help
+	  Enable Object Transfer Service.
+
+if BT_GATT_OTS
+
+config BT_GATT_OTS_MAX_INST_CNT
+	int "Maximum number of available OTS instances"
+	default 1
+
+config BT_GATT_OTS_MAX_OBJ_CNT
+	int "Maximum number of objects that server can store"
+	default 5
```

Comment:
It is per instance, perhaps I should add this in the description.

Defining a common object pool that is shared between instances could be added later on as another improvement. It could reduce the amount of memory required by this library in a multi-instance context when the number of objects, stored in each instance, is different.